dremio/repository.yaml

```diff
@@ -0,0 +1,8 @@
+name: dremio
+description: dremio deployed on plural
+category: DATA
+private: true
+icon: plural/icons/dremio.png
```

Comment:
actually just realized, an icon is needed here for the catalog listing

Reply:
Added the dremio icon @michaeljguarino